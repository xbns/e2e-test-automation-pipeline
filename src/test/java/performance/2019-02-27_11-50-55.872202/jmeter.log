2019-02-27 11:54:15,366 INFO o.a.j.u.JMeterUtils: Setting Locale to en_US
2019-02-27 11:54:15,378 INFO o.a.j.JMeter: Loading user properties from: /home/josphat/.bzt/jmeter-taurus/5.0/bin/user.properties
2019-02-27 11:54:15,378 INFO o.a.j.JMeter: Loading system properties from: /home/josphat/.bzt/jmeter-taurus/5.0/bin/system.properties
2019-02-27 11:54:15,378 INFO o.a.j.JMeter: Loading additional properties from: /home/josphat/Documents/TULAA/E2E-AUTOMATION-PIPELINE/blazemeter.com/e2e-test-automation-pipeline/src/test/java/performance/2019-02-27_11-50-55.872202/jmeter-bzt.properties
2019-02-27 11:54:15,379 INFO o.a.j.JMeter: Setting System properties from file: /home/josphat/Documents/TULAA/E2E-AUTOMATION-PIPELINE/blazemeter.com/e2e-test-automation-pipeline/src/test/java/performance/2019-02-27_11-50-55.872202/system.properties
2019-02-27 11:54:15,383 INFO o.a.j.JMeter: Copyright (c) 1998-2018 The Apache Software Foundation
2019-02-27 11:54:15,383 INFO o.a.j.JMeter: Version 5.0 r1840935
2019-02-27 11:54:15,383 INFO o.a.j.JMeter: java.version=1.8.0_191
2019-02-27 11:54:15,383 INFO o.a.j.JMeter: java.vm.name=OpenJDK 64-Bit Server VM
2019-02-27 11:54:15,383 INFO o.a.j.JMeter: os.name=Linux
2019-02-27 11:54:15,383 INFO o.a.j.JMeter: os.arch=amd64
2019-02-27 11:54:15,383 INFO o.a.j.JMeter: os.version=4.15.0-45-generic
2019-02-27 11:54:15,383 INFO o.a.j.JMeter: file.encoding=UTF-8
2019-02-27 11:54:15,383 INFO o.a.j.JMeter: Max memory     =1836580864
2019-02-27 11:54:15,384 INFO o.a.j.JMeter: Available Processors =4
2019-02-27 11:54:15,387 INFO o.a.j.JMeter: Default Locale=English (United States)
2019-02-27 11:54:15,387 INFO o.a.j.JMeter: JMeter  Locale=English (United States)
2019-02-27 11:54:15,388 INFO o.a.j.JMeter: JMeterHome=/home/josphat/.bzt/jmeter-taurus/5.0
2019-02-27 11:54:15,388 INFO o.a.j.JMeter: user.dir  =/home/josphat/Documents/TULAA/E2E-AUTOMATION-PIPELINE/blazemeter.com/e2e-test-automation-pipeline/src/test/java/performance
2019-02-27 11:54:15,388 INFO o.a.j.JMeter: PWD       =/home/josphat/Documents/TULAA/E2E-AUTOMATION-PIPELINE/blazemeter.com/e2e-test-automation-pipeline/src/test/java/performance
2019-02-27 11:54:15,388 INFO o.a.j.JMeter: IP: 127.0.1.1 Name: josphat-pc FullName: josphat-pc
2019-02-27 11:54:15,388 INFO o.a.j.JMeter: user.classpath=/home/josphat/Documents/TULAA/E2E-AUTOMATION-PIPELINE/blazemeter.com/e2e-test-automation-pipeline/src/test/java/performance/2019-02-27_11-50-55.872202:/home/josphat/Documents/TULAA/E2E-AUTOMATION-PIPELINE/blazemeter.com/e2e-test-automation-pipeline/src/test/java/performance/2019-02-27_11-50-55.872202
2019-02-27 11:54:15,388 INFO o.a.j.JMeter: Adding to classpath and loader: /home/josphat/Documents/TULAA/E2E-AUTOMATION-PIPELINE/blazemeter.com/e2e-test-automation-pipeline/src/test/java/performance/2019-02-27_11-50-55.872202
2019-02-27 11:54:15,389 INFO o.a.j.JMeter: Adding to classpath and loader: /home/josphat/Documents/TULAA/E2E-AUTOMATION-PIPELINE/blazemeter.com/e2e-test-automation-pipeline/src/test/java/performance/2019-02-27_11-50-55.872202
2019-02-27 11:54:15,392 INFO o.a.j.s.FileServer: Default base='/home/josphat/Documents/TULAA/E2E-AUTOMATION-PIPELINE/blazemeter.com/e2e-test-automation-pipeline/src/test/java/performance'
2019-02-27 11:54:15,397 INFO o.a.j.s.FileServer: Set new base='/home/josphat/Documents/TULAA/E2E-AUTOMATION-PIPELINE/blazemeter.com/e2e-test-automation-pipeline/src/test/java/performance/2019-02-27_11-50-55.872202'
2019-02-27 11:54:15,521 INFO o.a.j.s.SaveService: Testplan (JMX) version: 2.2. Testlog (JTL) version: 2.2
2019-02-27 11:54:15,531 INFO o.a.j.s.SaveService: Using SaveService properties file encoding UTF-8
2019-02-27 11:54:15,535 INFO o.a.j.s.SaveService: Using SaveService properties version 5.0
2019-02-27 11:54:15,540 INFO o.a.j.s.SaveService: Loading file: /home/josphat/Documents/TULAA/E2E-AUTOMATION-PIPELINE/blazemeter.com/e2e-test-automation-pipeline/src/test/java/performance/2019-02-27_11-50-55.872202/modified_requests.jmx
2019-02-27 11:54:15,583 INFO o.a.j.p.h.c.CacheManager: Will only cache the following methods: [GET]
2019-02-27 11:54:15,585 INFO o.a.j.p.h.c.CookieManager: Settings: Delete null: true Check: true Allow variable: true Save: false Prefix: COOKIE_
2019-02-27 11:54:15,600 INFO o.a.j.p.h.s.HTTPSamplerBase: Parser for text/html is org.apache.jmeter.protocol.http.parser.LagartoBasedHtmlParser
2019-02-27 11:54:15,600 INFO o.a.j.p.h.s.HTTPSamplerBase: Parser for application/xhtml+xml is org.apache.jmeter.protocol.http.parser.LagartoBasedHtmlParser
2019-02-27 11:54:15,600 INFO o.a.j.p.h.s.HTTPSamplerBase: Parser for application/xml is org.apache.jmeter.protocol.http.parser.LagartoBasedHtmlParser
2019-02-27 11:54:15,600 INFO o.a.j.p.h.s.HTTPSamplerBase: Parser for text/xml is org.apache.jmeter.protocol.http.parser.LagartoBasedHtmlParser
2019-02-27 11:54:15,600 INFO o.a.j.p.h.s.HTTPSamplerBase: Parser for text/vnd.wap.wml is org.apache.jmeter.protocol.http.parser.RegexpHTMLParser
2019-02-27 11:54:15,600 INFO o.a.j.p.h.s.HTTPSamplerBase: Parser for text/css is org.apache.jmeter.protocol.http.parser.CssParser
2019-02-27 11:54:15,634 INFO o.a.j.e.StandardJMeterEngine: Running the test!
2019-02-27 11:54:15,634 INFO o.a.j.s.SampleEvent: List of sample_variables: []
2019-02-27 11:54:15,635 INFO o.a.j.s.SampleEvent: List of sample_variables: []
2019-02-27 11:54:15,639 INFO o.a.j.e.u.CompoundVariable: Note: Function class names must contain the string: '.functions.'
2019-02-27 11:54:15,640 INFO o.a.j.e.u.CompoundVariable: Note: Function class names must not contain the string: '.gui.'
2019-02-27 11:54:16,250 INFO o.a.j.JMeter: Running test (1551257656250)
2019-02-27 11:54:16,289 INFO o.a.j.e.StandardJMeterEngine: Starting ThreadGroup: 1 : basic-performance-test
2019-02-27 11:54:16,289 INFO o.a.j.e.StandardJMeterEngine: Starting 10 threads for group basic-performance-test.
2019-02-27 11:54:16,289 INFO o.a.j.e.StandardJMeterEngine: Thread will continue on error
2019-02-27 11:54:16,294 INFO o.a.j.e.StandardJMeterEngine: All thread groups have been started
2019-02-27 11:54:16,294 INFO c.b.j.t.c.ConcurrencyThreadStarter: Start supplying threads
2019-02-27 11:54:19,312 INFO o.a.j.t.JMeterThread: Thread started: basic-performance-test 1-1
2019-02-27 11:54:19,313 INFO o.a.j.s.SampleResult: Note: Sample TimeStamps are START times
2019-02-27 11:54:19,313 INFO o.a.j.s.SampleResult: sampleresult.default.encoding is set to UTF-8
2019-02-27 11:54:19,313 INFO o.a.j.s.SampleResult: sampleresult.useNanoTime=true
2019-02-27 11:54:19,314 INFO o.a.j.s.SampleResult: sampleresult.nanoThreadSleep=5000
2019-02-27 11:54:20,110 INFO o.a.j.p.h.s.HTTPHCAbstractImpl: Local host = josphat-pc
2019-02-27 11:54:20,116 INFO o.a.j.p.h.s.HTTPHC4Impl: HTTP request retry count = 0
2019-02-27 11:54:21,224 INFO o.a.j.p.h.p.BaseParser: Created org.apache.jmeter.protocol.http.parser.LagartoBasedHtmlParser
2019-02-27 11:54:21,263 INFO o.a.j.p.h.s.ResourcesDownloader: Creating ResourcesDownloader with keepalive_inseconds : 60
2019-02-27 11:54:21,842 INFO o.a.j.p.h.p.BaseParser: Created org.apache.jmeter.protocol.http.parser.CssParser
2019-02-27 11:54:23,481 INFO o.a.j.u.JsseSSLManager: Using default SSL protocol: TLS
2019-02-27 11:54:23,481 INFO o.a.j.u.JsseSSLManager: SSL session context: per-thread
2019-02-27 11:54:25,318 INFO o.a.j.t.JMeterThread: Thread started: basic-performance-test 1-2
2019-02-27 11:54:31,328 INFO o.a.j.t.JMeterThread: Thread started: basic-performance-test 1-3
2019-02-27 11:54:37,326 INFO o.a.j.t.JMeterThread: Thread started: basic-performance-test 1-4
2019-02-27 11:54:43,329 INFO o.a.j.t.JMeterThread: Thread started: basic-performance-test 1-5
2019-02-27 11:54:49,332 INFO o.a.j.t.JMeterThread: Thread started: basic-performance-test 1-6
2019-02-27 11:54:55,336 INFO o.a.j.t.JMeterThread: Thread started: basic-performance-test 1-7
2019-02-27 11:55:01,338 INFO o.a.j.t.JMeterThread: Thread started: basic-performance-test 1-8
2019-02-27 11:55:07,341 INFO o.a.j.t.JMeterThread: Thread started: basic-performance-test 1-9
2019-02-27 11:55:13,344 INFO o.a.j.t.JMeterThread: Thread started: basic-performance-test 1-10
2019-02-27 11:56:46,366 INFO c.b.j.t.c.ConcurrencyThreadStarter: Done supplying threads
2019-02-27 11:56:46,898 INFO o.a.j.t.JMeterThread: Thread is done: basic-performance-test 1-2
2019-02-27 11:56:46,898 INFO o.a.j.t.JMeterThread: Thread finished: basic-performance-test 1-2
2019-02-27 11:56:46,996 INFO o.a.j.t.JMeterThread: Thread is done: basic-performance-test 1-3
2019-02-27 11:56:46,996 INFO o.a.j.t.JMeterThread: Thread finished: basic-performance-test 1-3
2019-02-27 11:56:47,059 INFO o.a.j.t.JMeterThread: Thread is done: basic-performance-test 1-7
2019-02-27 11:56:47,059 INFO o.a.j.t.JMeterThread: Thread finished: basic-performance-test 1-7
2019-02-27 11:56:47,318 INFO o.a.j.t.JMeterThread: Thread is done: basic-performance-test 1-9
2019-02-27 11:56:47,318 INFO o.a.j.t.JMeterThread: Thread finished: basic-performance-test 1-9
2019-02-27 11:56:47,744 INFO o.a.j.t.JMeterThread: Thread is done: basic-performance-test 1-5
2019-02-27 11:56:47,744 INFO o.a.j.t.JMeterThread: Thread finished: basic-performance-test 1-5
2019-02-27 11:56:48,146 INFO o.a.j.t.JMeterThread: Thread is done: basic-performance-test 1-1
2019-02-27 11:56:48,147 INFO o.a.j.t.JMeterThread: Thread finished: basic-performance-test 1-1
2019-02-27 11:56:48,247 INFO o.a.j.t.JMeterThread: Thread is done: basic-performance-test 1-6
2019-02-27 11:56:48,248 INFO o.a.j.t.JMeterThread: Thread finished: basic-performance-test 1-6
2019-02-27 11:56:48,409 INFO o.a.j.t.JMeterThread: Thread is done: basic-performance-test 1-4
2019-02-27 11:56:48,409 INFO o.a.j.t.JMeterThread: Thread finished: basic-performance-test 1-4
2019-02-27 11:56:48,569 INFO o.a.j.t.JMeterThread: Thread is done: basic-performance-test 1-10
2019-02-27 11:56:48,569 INFO o.a.j.t.JMeterThread: Thread finished: basic-performance-test 1-10
2019-02-27 11:56:48,663 INFO o.a.j.t.JMeterThread: Thread is done: basic-performance-test 1-8
2019-02-27 11:56:48,663 INFO o.a.j.t.JMeterThread: Thread finished: basic-performance-test 1-8
2019-02-27 11:56:48,664 INFO o.a.j.e.StandardJMeterEngine: Notifying test listeners of end of test
2019-02-27 11:56:48,666 INFO o.a.j.e.StandardJMeterEngine: Forced JVM shutdown requested at end of test
