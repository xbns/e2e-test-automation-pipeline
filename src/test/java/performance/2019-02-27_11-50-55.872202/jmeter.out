Created the tree successfully using /home/josphat/Documents/TULAA/E2E-AUTOMATION-PIPELINE/blazemeter.com/e2e-test-automation-pipeline/src/test/java/performance/2019-02-27_11-50-55.872202/modified_requests.jmx
Starting the test @ Wed Feb 27 11:54:15 EAT 2019 (1551257655633)
Waiting for possible Shutdown/StopTestNow/Heapdump message on port 4445
Tidying up ...    @ Wed Feb 27 11:56:48 EAT 2019 (1551257808665)
... end of run
